build.rs

```diff
@@ -167,11 +181,11 @@ fn write_testsuite_tests(
 
 /// Ignore tests that aren't supported yet.
 fn ignore(testsuite: &str, name: &str, strategy: &str) -> bool {
-    let is_multi_value = testsuite.ends_with("multi-value");
+    let is_multi_value = testsuite.ends_with("multi_value");
```

Comment:
In my submodule, the path is `spec_testsuite/proposals/multi-value` with a hyphen, not an underscore. Was this change and the similar one elsewhere intentional?